tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -1186,3 +1190,220 @@ def test_log_artifacts_provides_failure_info(self, databricks_artifact_repo, tmp
             assert "MOCK ERROR 1" in err_msg
             assert "file_2.txt" in err_msg
             assert "MOCK ERROR 2" in err_msg
+
+
+def test_multipart_upload(databricks_artifact_repo, tmp_path):
+    size = 250 * 1000 * 1000  # 250 MB -> requires 3 part uploads
```

Comment:
to make the test running faster, we can mock `_MULTIPART_UPLOAD_CHUNK_SIZE` to be a small number.

Reply:
0.17s is fast enough, but we can still save disk space the temp data uses.

Reply:
@WeichenXu123 

> 0.17s is fast enough, but we can still save disk space the temp data uses.

Why do we want to sae the disk space?

Reply:
For GitHub aciton it is fine.
but we might run test locally and if test crashes it might leave temp file on local disk. If we rerun test multiple times locally it might still eat a lot of disk space.

Reply:
See https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-default-base-temporary-directory